daffy_lib.gemspec

```diff
@@ -18,16 +18,26 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+  spec.add_development_dependency 'attr_encrypted'
   spec.add_development_dependency 'bundler'
   spec.add_development_dependency 'bundler-audit'
   spec.add_development_dependency 'codecov'
+  spec.add_development_dependency 'factory_bot_rails'
+  spec.add_development_dependency 'porky_lib'
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rspec'
+  spec.add_development_dependency 'rspec-collection_matchers'
+  spec.add_development_dependency 'rspec-mocks'
+  spec.add_development_dependency 'rspec-rails'
   spec.add_development_dependency 'rspec_junit_formatter'
   spec.add_development_dependency 'rubocop'
   spec.add_development_dependency 'rubocop-performance'
   spec.add_development_dependency 'rubocop-rspec'
   spec.add_development_dependency 'rubocop_runner'
   spec.add_development_dependency 'simplecov'
+  spec.add_development_dependency 'sqlite3'
   spec.add_development_dependency 'timecop'
+
+  spec.add_dependency 'rails'
+  spec.add_dependency 'redis'
```

Comment:
This is an artifact from this code and its tests being originally put into Roadrunner.  

Could look into catching/testing generic errors later, but since Redis is now built into Rails it might be ok to leave them for now and just say this gem is for redis-backed dbs.